### PR TITLE
updated the grindercode so now it does more

### PR DESCRIPTION
**Pull request by iampei**
This is for the exercise part 3.

### CODE REVIEW

**Comment by iampei** [comment]
Add the fuzzbuzz code

**Comment by PythonCHB** [comment]
was "grinder" code a auto-correct?

File: students/pei/session02/fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+for i in range (1, 101):
```
Comment (PythonCHB) [inline students/pei/session02/fizzbuzz.py]: maybe see if you can do this without checking for i%3 and i%5 twice...

File: students/pei/session02/grid_printer.py [hunk 2]
```diff
@@ -3,9 +3,21 @@
...
-def mkgrid(k):
-    a = ('+' + '-'*4) * 2 + '+'
-    b = ('|' + " "*4) * 2 + '|'
-    print ((a + ('\n' + b)*k) + '\n' + (a + ('\n' + b)*k) + '\n' +a)
+#def mkgrid(k):
+    #a = ('+' + '-'*(4+k)) * 2 + '+'
+    #b = ('|' + " "*(4+k)) * 2 + '|'
+    #print ((a + ('\n' + b)*k) + '\n' + (a + ('\n' + b)*k) + '\n' +a)
+
+def mkgrid2(k, j):
```
Comment (PythonCHB) [inline students/pei/session02/grid_printer.py]: nicely done!

**Comment by PythonCHB** [comment]
there were conflicts -- not sure why... but if you do a:

```
git pull upstream master
```
you should get the final version

**Pull request merged**